WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormDataModel.swift

```diff
@@ -35,6 +33,8 @@ protocol ProductFormDataModel {
     var dimensions: ProductDimensions { get }
     var shippingClass: String? { get }
     var shippingClassID: Int64 { get }
+    // Whether shipping settings are available for the product.
+    func isShippingEnabled() -> Bool
```

Comment:
Just curious, why are these methods instead of properties? Was it to distinguish them from the DB properties? Is my question confusing? 🤪

Reply:
Yes I was trying to distinguish the visibility logic from the DB properties, it might be better if they are in two separate protocols but trying to keep the changes not too large for now 🙂